frontends/qt/server/server.go

```diff
@@ -130,6 +130,9 @@ func serve(
 			},
 			DeviceInfosFunc: usb.DeviceInfos,
 			SystemOpenFunc:  system.Open,
+			UsingMobileData: func() bool {
+				return false
```

Comment:
I suppose a user could also be on a tethered connection over a phone but there's not much we can do, right.

Reply:
I guess not.